### PR TITLE
EG-465 hashLookup returns "No matching transaction found" for any transaction hash - Updated command to support transaction id-s as well (not just hashes of transactions)

### PR DESCRIPTION
**Pull request by pnemeth**
- Improved functionality to support transaction id-s not only hashes of transactions
- Updated test
- Update manual

### CODE REVIEW

File: tools/shell/src/integration-test/kotlin/net/corda/tools/shell/HashLookupCommandTest.kt [hunk 1]
```diff
@@ -35,29 +34,30 @@ class HashLookupCommandTest {
...
+    private fun testCommand(user: User, node: NodeHandle, command: String, expected: String) {
+        val session = connectToShell(user, node)
         val channel = session.openChannel("exec") as ChannelExec
         channel.setCommand(command)
         channel.connect(5000)
 
         assertTrue(channel.isConnected)
 
         val response = String(Streams.readAll(channel.inputStream))
+        println(response)
```
Comment (JamesHR3) [inline tools/shell/src/integration-test/kotlin/net/corda/tools/shell/HashLookupCommandTest.kt]: I don't think you want to leave this in.

File: tools/shell/src/main/java/net/corda/tools/shell/HashLookupShellCommand.java [hunk 2]
```diff
@@ -19,22 +19,21 @@
...
-    @Man("Checks if a transaction matching a specified Id hash value is recorded on this node.\n\n" +
+    final private String manualText ="Checks if a transaction matching a specified Id hash value is recorded on this node.\n\n" +
+            "Both the transaction Id and the hashed value of a transaction Id (as returned by the Notary in case of a double-spend) is a valid input.\n" +
             "This is mainly intended to be used for troubleshooting notarisation issues when a\n" +
             "state is claimed to be already consumed by another transaction.\n\n" +
-            "Example usage: hashLookup E470FD8A6350A74217B0A99EA5FB71F091C84C64AD0DE0E72ECC10421D03AAC9"
-    )
+            "Example usage: hashLookup E470FD8A6350A74217B0A99EA5FB71F091C84C64AD0DE0E72ECC10421D03AAC9";
+
+    @Command
```
Comment (JamesHR3) [inline tools/shell/src/main/java/net/corda/tools/shell/HashLookupShellCommand.java:28]: You should also edit the `@Usage` statement below.

**Review by JamesHR3 — CHANGES_REQUESTED** [review]
Couple of comments.

Could you put the Jira ticket as the first item in the PR title? It's generally good style to do this, and I'm not sure if Jira picks the PR up if you don't. It's useful to put a description in the title for exactly what it is you've fixed here too, partially for people browsing PRs to figure out if they care, and partly as the PR title often ends up as the message in `git log` for the change.

**Review by JamesHR3 — APPROVED** [review]
LGTM

**Comment by JamesHR3** [comment]
@adagys do you know why the test here was `@Ignore`d? I just want to make sure we're not going to introduce further TC instability or similar by enabling this.

**Comment by adagys** [comment]
@JamesHR3 Unfortunately, I don't remember why

**Pull request merged**